src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -193,20 +193,25 @@ public static Dictionary<string, OrganelleEfficiency> ComputeOrganelleProcessEff
     ///   Computes the energy balance for the given organelles in biome and at a given time during the day (or type
     ///   can be specified to be a different type of value)
     /// </summary>
-    public static EnergyBalanceInfo ComputeEnergyBalance(IReadOnlyList<OrganelleTemplate> organelles,
+    public static EnergyBalanceInfoFull ComputeEnergyBalance(IReadOnlyList<OrganelleTemplate> organelles,
         IBiomeConditions biome, MembraneType membrane, bool includeMovementCost, bool isPlayerSpecies,
         WorldGenerationSettings worldSettings, CompoundAmountType amountType, bool calculateRequiredResources)
     {
         var organellesList = organelles.ToList();
 
         var maximumMovementDirection = MicrobeInternalCalculations.MaximumSpeedDirection(organellesList);
-        return ComputeEnergyBalance(organellesList, biome, membrane, maximumMovementDirection, includeMovementCost,
+        EnergyBalanceInfoFull result = new();
+        ComputeEnergyBalance(result, organellesList, biome, membrane, maximumMovementDirection, includeMovementCost,
             isPlayerSpecies, worldSettings, amountType, calculateRequiredResources, null);
+        return result;
     }
 
     /// <summary>
     ///   Computes the energy balance for the given organelles in biome
     /// </summary>
+    /// <param name="result">
+    ///   The energy balance info that is returned. This may be an EnergyBalanceSimple or an EnergyBalanceFull.
```

Comment:
`<see cref=""/>` should be used when referring to code in the XML to make it navigable and Jetbrains will check to make sure the reference is valid.